src/scss/custom/components/_footer.scss

```diff
@@ -0,0 +1,164 @@
+footer {
```

Comment:
I think we should use a class instead of putting style directly on all `footer` elements. Any site that uses this library will have to override these rules for other uses of `footer`.